### PR TITLE
Update dashboards for multi-cluster monitoring

### PR DESCRIPTION
**Pull request by michal-gubricky**

### CODE REVIEW

File: helpers/values-multicluster.yaml [hunk 1]
```diff
@@ -78,6 +97,12 @@ clusterMonitoring:
 
 templates:
   L1:
+    host:
```
Comment (chess-knight) [inline helpers/values-multicluster.yaml:100]: See https://github.com/dNationCloud/kubernetes-monitoring/pull/153/files#r1013701505

File: helpers/values-multicluster.yaml [hunk 2]
```diff
@@ -26,10 +26,29 @@ containerLabel: &containerLabel
...
+    jobName: host-01
+    host:
+      address: 95.217.178.89
+    serviceMonitor:
+      endpoints:
+      - port: "9100"  # Node exporter port
+        interval: 30s
+        path: /metrics
+
 clusterMonitoring:
```
Comment (chess-knight) [inline helpers/values-multicluster.yaml:43]: Can you also please update helpers/README.md and helpers/FAQ.md?

Comment (chess-knight) [reply helpers/values-multicluster.yaml:43]: And what about GETTING_STARTED.md and README.md, do they need editing too?

File: jsonnet/dashboards/hosts/alert-overview.libsonnet [hunk 3]
```diff
@@ -16,37 +16,57 @@
...
+            { alias: 'warningCode', pattern: 'Value #A', type: 'number', thresholds: warning_thresholds, colors: colors, colorMode: 'row'},
+            { alias: 'criticalCode  ', pattern: 'Value #B', type: 'number', thresholds: critical_thresholds, colors: colors, colorMode: 'row'},
           ]
         )
-        .addTarget({ type: 'table', expr: 'ALERTS{alertname!="Watchdog", severity=~"$severity", alertgroup=~"$alertgroup", job=~"$job"}' });
+        .addTargets([
+          prometheus.target('ALERTS{job=~"$job", alertname!="Watchdog", alertstate=~"firing|pending", severity="warning", severity=~"$severity", alertgroup=~"$alertgroup"} * 2', format='table', instant=true),
+          prometheus.target('ALERTS{job=~"$job", alertname!="Watchdog", alertstate=~"firing|pending", severity="critical", severity=~"$severity", alertgroup=~"$alertgroup"} * 3', format='table', instant=true),
+        ])
+//        .addTarget({ type: 'table', expr: 'ALERTS{alertname!="Watchdog", severity=~"$severity", alertgroup=~"$alertgroup", job=~"$job"}' })
```
Comment (chess-knight) [inline jsonnet/dashboards/hosts/alert-overview.libsonnet:49]:
```suggestion
```

File: jsonnet/dashboards/k8s/k8s-monitoring.libsonnet [hunk 4]
```diff
@@ -55,12 +55,21 @@ local getGridY(offset, index, panelWidth, panelHeight) =
...
+
       local explorerLink =
         link.dashboards(
           title='Logs',
           tags=[],
           icon='doc',
-          url='/explore?orgId=1&left=%5B%22now-7d%22,%22now%22,%22$datasource_logs%22,%7B%22expr%22:%22%7Bnamespace%3D%5C%22kube-system%5C%22,%20stream%3D%5C%22stderr%5C%22%7D%20%7C~%20%5C%22(%3Fi)error%5C%22%20!~%20%5C%22Final%20error%20received,%20removing%20PVC%20.%2B%20from%20claims%20in%20progress%5C%22%22%7D,%7B%22mode%22:%22Logs%22%7D,%7B%22ui%22:%5Btrue,true,true,%22numbers%22%5D%7D%5D',
+          #url='/explore?orgId=1&left=%5B%22now-7d%22,%22now%22,%22$datasource_logs%22,%7B%22expr%22:%22%7Bnamespace%3D%5C%22kube-system%5C%22,%20stream%3D%5C%22stderr%5C%22%7D%20%7C~%20%5C%22(%3Fi)error%5C%22%20!~%20%5C%22Final%20error%20received,%20removing%20PVC%20.%2B%20from%20claims%20in%20progress%5C%22%22%7D,%7B%22mode%22:%22Logs%22%7D,%7B%22ui%22:%5Btrue,true,true,%22numbers%22%5D%7D%5D',
```
Comment (chess-knight) [inline jsonnet/dashboards/k8s/k8s-monitoring.libsonnet:71]:
```suggestion
```

File: jsonnet/dashboards/monitoring.libsonnet [hunk 5]
```diff
@@ -181,10 +185,13 @@ local getClusterRowGridY(numOfClusters, panelWidth, panelHeight) =
             .addThresholds($.grafanaThresholds(tpl.panel.thresholds))
             .addMappings(tpl.panel.mappings)
             .addDataLinks(
-              if std.length(tpl.panel.dataLinks) > 0 then
-                tpl.panel.dataLinks
-              else
-                [{ title: 'Kubernetes Monitoring', url: '/d/%s?%s' % [getUid($._config.grafanaDashboards.ids.k8sMonitoring, cluster, $._config.templates.L1.k8s), dataLinkCommonArgs] }]
+              #TODO  updateDataLinksCommonArgs probably doesnt have to be here
```
Comment (chess-knight) [inline jsonnet/dashboards/monitoring.libsonnet:188]: Is this TODO relevant?

File: jsonnet/dashboards/monitoring.libsonnet [hunk 6]
```diff
@@ -134,12 +136,14 @@ local getClusterRowGridY(numOfClusters, panelWidth, panelHeight) =
...
+
             // multiple cluster monitoring isn't supported yet, replace lines when adding support for multiple clusters
-            local dataLinkCommonArgs = $._config.grafanaDashboards.dataLinkCommonArgs;
-            //local dataLinkCommonArgs = std.strReplace($._config.grafanaDashboards.dataLinkCommonArgs, '$cluster|', cluster.name);
+            //local dataLinkCommonArgs = $._config.grafanaDashboards.dataLinkCommonArgs;
+            local dataLinkCommonArgs = std.strReplace($._config.grafanaDashboards.dataLinkCommonArgs, '$cluster|', clusterLabel);
 
             // when multiple cluster will be supported, cluster variable will in expr will be cluster name
-            local localCluster = { name: '' };
+            //local localCluster = { name: '' };
```
Comment (chess-knight) [inline jsonnet/dashboards/monitoring.libsonnet:146]: Please remove unnecessary lines/comments.

File: helpers/FAQ.md [hunk 7]
```diff
@@ -12,8 +12,10 @@ When some threshold is crossed and this state  persists for **5** minutes (defau
 The orange state panel  with `Warning` label is displayed in case of warning alert. Red state panel with  `Critical` label is displayed in case of critical alert.
 We have implemented intuitive green, orange and red color indicators that are signalizing if your action is needed or if everything is OK.
 If you want to see more information about your k8s cluster, just drill down by left-clicking on
-the relevant state panel. As of now, only single cluster monitoring is supported. Multi cluster monitoring support is under development.
+the relevant state panel. As of now, only single cluster monitoring is supported. Multi-cluster monitoring support is avaible.
```
Comment (chess-knight) [inline helpers/FAQ.md:15]: Please repair this sentence. `only single cluster monitoring is supported`

File: helpers/README.md [hunk 8]
```diff
@@ -56,6 +56,8 @@ helm install dnation-kubernetes-monitoring chart --set releaseOverride=dnation-k
 helm install dnation-kubernetes-monitoring chart --set releaseOverride=dnation-kubernetes-monitoring-stack -f helpers/values-cluster-elk.yaml
 # Host monitoring example
 helm install dnation-kubernetes-monitoring chart --set releaseOverride=dnation-kubernetes-monitoring-stack -f helpers/values-host.yaml
+# Multi-cluster monitoring example
+helm install dnation-kubernetes-monitoring chart --set releaseOverride=dnation-kubernetes-monitoring-stack -f helpers/values-multicluster.yaml
```
Comment (chess-knight) [inline helpers/README.md:60]: You renamed this file but `values-cluster.yaml` are still in this instructions.

File: jsonnet/config.libsonnet [hunk 9]
```diff
@@ -153,14 +154,112 @@
       alertGroupHost: 'Host',
       alertGroupHostApp: 'HostApp',
     },
-    //multiple cluster monitoring isn't supported yet
+
     clusterMonitoring: {
       enabled: true,
-      clusters: [],
+ //     clusters: [{name: "Workload cluster", description: ""}],
+      clusters: [
```
[the remark this answers is not in the document]
Comment (chess-knight) [reply jsonnet/config.libsonnet:160]: i think it should be removed from the default jsonnet config, we do overrides trough [values](https://github.com/dNationCloud/kubernetes-monitoring/blob/main/chart/values.yaml#L81)

File: jsonnet/dashboards/monitoring.libsonnet (hunk 5 above)
Comment (michal-gubricky) [reply jsonnet/dashboards/monitoring.libsonnet]: It is not relevant

**Review by chess-knight — APPROVED** [review]

**Pull request merged**